recipes/ctrack/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,20 @@
+#include "ctrack.hpp"
+
+void expensiveOperation() {
+    CTRACK;
+    // Simulating some work
+    for (int i = 0; i < 1000000; ++i) {
+        // Do something
+    }
+}
+
+int main() {
+    for (int i = 0; i < 100; ++i) {
+        expensiveOperation();
+    }
```

Comment:
```suggestion
#include "ctrack.hpp"
#include <iostream>

void expensiveOperation() {
    CTRACK;
    // Simulating some work
    for (int i = 0; i < 5; ++i) {
        std::cout << i << " ";
    }
    std::cout << std::endl;
}

int main() {
    for (int i = 0; i < 2; ++i) {
        expensiveOperation();
    }
```

Else this might get compiled to nothing!